File: Assignments/2_Operators and Expression/Javascript/Solutions/Q1.js
// Q1.js
// Program to compute mean and median of 10 numbers

const prompt = require('prompt-sync')();

let numbers = [];
for (let i = 0; i < 10; i++) {
  numbers.push(parseFloat(prompt(`Enter number ${i + 1}: `)));
}

numbers.sort((a, b) => a - b);

const mean = numbers.reduce((acc, val) => acc + val, 0) / numbers.length;
const median = (numbers[4] + numbers[5]) / 2;

console.log(`Mean: ${mean}`);
console.log(`Median: ${median}`);

File: Assignments/2_Operators and Expression/C#/Q1.cs
using System;
using System.Linq;

class Program
{
    static void Main()
    {
        int[] numbers = new int[10];
        Console.WriteLine("Enter 10 numbers in sorted order:");
        for (int i = 0; i < 10; i++)
        {
            numbers[i] = int.Parse(Console.ReadLine());
        }

        double mean = numbers.Average();
        double median = (numbers[4] + numbers[5]) / 2.0;

        Console.WriteLine("Mean: " + mean);
        Console.WriteLine("Median: " + median);
    }
}
